doc/mk_sdcard_image.sh

```diff
@@ -34,6 +37,7 @@ usage()
 	echo "  -u domain       Optional, unpack only specified domain: dom0, domd, domf, doma, domu"
 	echo "  -f              Optional, force rewrite of image file (useful for batch usage)"
 	echo "  -r              Optional, force repack of initiramfs and add DomA Image(from DomA dir) to it"
+	echo "  -v              Optional, create specific android config with Android verified boot cmdline"
```

Comment:
IMO, mixed functionality. The original intention of this script to flash images as they are without altering